src/__tests__/colors.spec.ts

```diff
@@ -48,6 +48,10 @@ describe(`colors`, () => {
     expect(tw`text-black`).toEqual({ color: `#000` });
   });
 
+  test(`tint colors`, () => {
+    expect(tw`tint-black`).toEqual({ tintColor: `#000` });
```

Comment:
can you add a couple more tests here, for stuff like:

- arbitrary values: `tint-[#eaeaea]`
- customized theme colors
- opacity: `tint-black/50`

they should all immediately pass, i would just feel better with the test coverage